src/media/js/addon/containers/reviewDetail.js

```diff
@@ -31,15 +33,26 @@ export class AddonReviewDetail extends React.Component {
   }
 
   render() {
-    if (!this.props.addon || !this.props.addon.slug) {
+    const addon = this.props.addon;
+
+    if (!addon || !addon.slug) {
```

Comment:
besides moving to a diff render method, figuring out a pattern for fragment loading placeholders will be more involved